boards/arm/mps2_an521/mps2_an521_nonsecure_defconfig

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2018-2019 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_ARM=y
+CONFIG_SOC_SERIES_MPS2=y
+CONFIG_SOC_MPS2_AN521=y
+CONFIG_BOARD_MPS2_AN521=y
+CONFIG_ARM_TRUSTZONE_M=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_RUNTIME_NMI=y
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+CONFIG_ARM_MPU=y
```

Comment:
Why is the MPU not enabled also in the secure board? (Maybe it's somewhere there, but I couldn't find it)